guides/routing.md

```diff
@@ -679,22 +685,22 @@ end
 
 This means the plugs in the `authenticate_user` and `ensure_admin` pipelines will be called before the `BackgroundJob.Plug` allowing them to send an appropriate response and halt the request accordingly.
 
-The `opts` that are received in the `init/1` callback of the Module Plug can be passed as a 3rd argument. For example, maybe the background job lets you set the name of your application to be displayed on the page. This could be passed with:
+The `opts` that are received in the `init/1` callback of the Module Plug can be passed as a third argument. For example, maybe the background job lets you set the name of your application to be displayed on the page. This could be passed with:
 
 ```elixir
 forward "/jobs", BackgroundJob.Plug, name: "Hello Phoenix"
 ```
 
 There is a fourth `router_opts` argument that can be passed. These options are outlined in the `Phoenix.Router.scope/2` documentation.
 
-`BackgroundJob.Plug` can be implemented as any Module Plug discussed [in the Plug guide](plug.html). Note though it is not advised to forward to another Phoenix endpoint. This is because plugs defined by your app and the forwarded endpoint would be invoked twice, which may lead to errors.
+`BackgroundJob.Plug` can be implemented as any module plug discussed in the [Plug guide](plug.html). Note though it is not advised to forward to another Phoenix endpoint. This is because plugs defined by your app and the forwarded endpoint would be invoked twice, which may lead to errors.
 
 ## Summary
 
 Routing is a big topic, and we have covered a lot of ground here. The important points to take away from this guide are:
 
 - Routes which begin with an HTTP verb name expand to a single clause of the match function.
-- Routes which begin with 'resources' expand to 8 clauses of the match function.
+- Routes which begin with `resources` expand to 8 clauses of the match function.
```

Comment:
Here is it not clear IMO in the above two listed items this thing about "Routes which begin"...
Should it be "routes declared with ..." ?